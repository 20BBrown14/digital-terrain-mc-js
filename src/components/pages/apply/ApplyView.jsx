import React from 'react';

/**
 * Apply page view.
 */
const ApplyView = () => (
  <div>
    Apply Page
  </div>
);

export default ApplyView;
