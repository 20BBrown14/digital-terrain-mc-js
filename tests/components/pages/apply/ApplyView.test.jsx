import React from 'react';
import ApplyView from '../../../../src/components/pages/apply/ApplyView';

describe('ApplyView', () => {
  it('Renders a default view', () => {
    const view = render(<ApplyView />);
    expect(view).toMatchSnapshot();
  });
});
