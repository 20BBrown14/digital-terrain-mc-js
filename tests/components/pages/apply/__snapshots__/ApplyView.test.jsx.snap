// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplyView Renders a default view 1`] = `
<div>
  Apply Page
</div>
`;
